deepchem/models/torch_models/tests/test_fno.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+try:
+    import torch
+    has_torch = True
+except ModuleNotFoundError:
+    has_torch = False
+    pass
+
+
+@pytest.mark.torch
+def test_fno_block_construction():
+    from deepchem.models.torch_models.fno import FNOBlock
```

Comment:
Minor comment: Can you please add a brief description of what the tests are doing?